docs/user/attestations/internals.md

```diff
@@ -0,0 +1,335 @@
+            }
+        ]
+    },
+    "envelope": {
```

Comment:
If this is coming from the bundle, should this be `dsse_envelope`?

Reply:
Same as above -- we called it just `envelope` for the attestation object since there's nothing other than DSSE that it can be, although I have no objection to renaming it either 🙂